contracts/VoucherHub.sol

```diff
@@ -172,7 +172,10 @@ contract VoucherHub is
         // The proxy contract does a delegatecall to its implementation.
         // Re-Entrancy safe because the target contract is controlled.
         Voucher(voucherAddress).initialize(owner, address(this), expiration, voucherType);
-        IERC20($._iexecPoco).transfer(voucherAddress, value); // SRLC
+        if (!IERC20($._iexecPoco).transfer(voucherAddress, value)) {
+            // SRLC
+            revert("VoucherHub: SRLC transfer to voucher failed");
+        }
```

Comment:
```suggestion
        // SRLC
        if (!IERC20($._iexecPoco).transfer(voucherAddress, value)) {
            revert("VoucherHub: SRLC transfer to voucher failed");
        }
```

Reply:
Doesn't Slither raise a warning and recommends making external calls at the very end?

Reply:
Good point, actually Slither did not mention this.